### ISSUE TITLE
MQTT client not connecting to IBM-IOT

### ISSUE THREAD

**Issue by pravallikapothuru** [IBM-Bluemix/parrot-ar-sample#3]
Hi,

Please help on this.
I have imported parrot-sample application from github.I have created device and generated api key,and kept these in drone config file.please find the attached config file.

when I am trying to run the parrot-drone-controller.js file.It was showing client closed,It is not connecting to IOT.I have attached error screenshot.

![error](https://user-images.githubusercontent.com/23692822/27850100-cc4b4cfa-616f-11e7-9587-546c1771e9ce.PNG)

![config](https://user-images.githubusercontent.com/23692822/27850142-0f6c3e90-6170-11e7-8bad-1cfb47ed7488.PNG)

**Comment by andrewyuau** [comment]
Hi, is your Windows machine connected to your AR Drone's wifi network? Try to disconnect other devices from your AR Drone too.

**Comment by pravallikapothuru** [comment]
Yes,machine connected to drons's wifi and drone not connected to other devices also at that time.

**Comment by andrewyuau** [comment]
How is your Windows machine connected to the Internet? Ethernet? You should at least see "MQTT client connected to IBM IoT Cloud."

**Comment by pravallikapothuru** [comment]
I am connecting to Ethernet.

**Comment by andrewyuau** [comment]
Can you ping both x29udg.messaging.internetofthings.ibmcloud.com and 192.168.1.1? The AR drone should be on 192.168.1.1.
